src/Transfer/Examples/YamlToEzPlatformContentType/resources/yaml/mini.yml

```diff
@@ -0,0 +1,6 @@
+banner:
+    fields:
+        title:
+            type: ezstring
+        description:
+            type: ezstring
```

Comment:
@htollefsen You can try either `title: []` or `title: {}` depending on whether you need an array or an object, respectively.